### PR TITLE
UI layover

### PR DESCRIPTION
**Pull request by kristianeboe**
image

image

image

### CODE REVIEW

File: src/components/AppHeader.tsx
Comment (kristianeboe) [inline src/components/AppHeader.tsx:140]: Looks like the mobile menu was gone for a while, this makes it appear again

File: src/components/CardGrid.tsx
Comment (kristianeboe) [inline src/components/CardGrid.tsx:75]: 4x4 is the new standard

File: src/components/ProfilePreviewCard.tsx
Comment (kristianeboe) [inline src/components/ProfilePreviewCard.tsx:54]: Various changes in this file to improve the "html flow" of the component

File: src/graphql/home/home.graphql
Comment (kristianeboe) [inline src/graphql/home/home.graphql:150]: Removes chance of one user clogging up the feed with mint events in the Hero section

File: src/graphql/profile/getProfileFollowerOverview.graphql
Comment (kristianeboe) [inline src/graphql/profile/getProfileFollowerOverview.graphql:23]: Moved the getCollectedBy.graqph query here since it made sense to centralize. We can probably delete the other query file

File: src/pages/alpha/index.tsx
Comment (kristianeboe) [inline src/pages/alpha/index.tsx:78]: This code is just moved around for memoization purposes

File: src/pages/profiles/[publicKey]/nfts.tsx
Comment (kristianeboe) [inline src/pages/profiles/[publicKey]/nfts.tsx:188]: Moved the primary button of NFT cards to its own row so it does not conflict with price or other information. It works better in the bigger grid size. Approved by Belle.

**Review by kristianeboe — COMMENTED** [review]
Tried to explain a few things here and there, but again, this is mostly cosmetic :) Looks good though ;)

**Review by austinmilt — CHANGES_REQUESTED** [review]
Explore removing the coupling of `CardGrid` with specific cases within the app (i.e. profiles)

File: src/graphql/profile/getProfileFollowerOverview.graphql
Comment (kristianeboe) [reply src/graphql/profile/getProfileFollowerOverview.graphql:23]: I lowered it to 300 now. My impression is that it's a maximum of 1000 that is returned though and most profiles have way less :)

File: src/pages/alpha/index.tsx
Comment (kristianeboe) [reply src/pages/alpha/index.tsx:82]: I'll hold off on this for now, but I left a todo comment with reference to useHolaplexInfiniteScrollQuery and useApolloQuery.ts

File: src/pages/discover/profiles.tsx
Comment (kristianeboe) [reply src/pages/discover/profiles.tsx:65]: Since we only have a limited number of card types I don't think it's nescecarily a bad idea to have the grid have some knowledge of the card type, but I defer to you and added a bigGridClassNameOverride prop instead.

**Pull request merged**
